### PR TITLE
Development

### PR DESCRIPTION
**Pull request by ATOMNFT**

### CODE REVIEW

File: minigotchi-ESP32/display.cpp [hunk 1]
```diff
@@ -62,59 +66,67 @@ void Display::startScreen() {
...
       delay(5);
-      tft.setTextSize(4); // Set text size
+      tft.setTextSize(4);  // Set text size
       delay(5);
-      tft.println(text); // Print text
+      tft.println(text);  // Print text
       delay(5);
-    } else if (Config::screen == "T_DISPLAY_S3") {
-      tft.fillScreen(TFT_BLACK);
+    } else if (Config::screen == "TTGO_T_DISPLAY") { // New condition for TTGO_T_DISPLAY
```
Comment (dj1ch) [inline minigotchi-ESP32/display.cpp:111]: im not sure if we have different displays, but I would prefer that we keep the settings I put down for it, I tried to adhere to the name of the product kind of. As for screen coordinates, I don't really care as long as it's *mostly* pwnagotchi-like and it looks similar to the other displays

File: minigotchi-ESP32/display.h [hunk 2]
```diff
@@ -1,34 +1,38 @@
...
 
 // Define display parameters for CYD
 #define CYD_SCREEN_WIDTH 240
 #define CYD_SCREEN_HEIGHT 320
 
-// lilygo t-display-t3
-#define T_DISPLAY_S3_WIDTH 320
-#define T_DISPLAY_S3_HEIGHT 170
+// Define display parameters for TTGO_T_DISPLAY
+#define TTGO_T_DISPLAY_WIDTH 135
```
Comment (dj1ch) [inline minigotchi-ESP32/display.h:30]: Do we have different displays? I got the information for the display size and stuff from the product page, but if they're different I think we should make them separate values

**Review by dj1ch — CHANGES_REQUESTED** [review]
Make sure you patch this with restyle, plus a couple other things

**Comment by ATOMNFT** [comment]
> Make sure you patch this with restyle, plus a couple other things

Need to read up on how to get this done.

File: minigotchi-ESP32/display.cpp (hunk 1 above)
Comment (ATOMNFT) [reply minigotchi-ESP32/display.cpp:111]: To make it simpler and more streamlined I would suggest it just be T-Display or something similar. As your sketch worked on my 1.14in t-display. Ppl will see the s3 and think it only runs on that.

**Review by ATOMNFT — COMMENTED** [review]
Redid Repo.

**Comment by dj1ch** [comment]
> Redid Repo.

Gotcha, let me know when you're ready

**Pull request closed without merging**